.travis.yml

```diff
@@ -10,12 +10,15 @@ git:
 
 before_install:
   - git submodule update --init --recursive
+install: bundle install
 
 jobs:
   include:
+    - stage: test
+      script: bundle exec rake test:ios 
+    - script: bundle exec rake test:macos
```

Comment:
What is the purpose of testing iOS & macOS here too? Seems like it's just doubling up on some of the test but not others... I know this comes from flovilmart's PR - maybe he was just using it for testing?

Reply:
It runs all test cases both iOS and macOS. 

We can get considering the separate it. (e.g. test-iOS and test-macOS)

But I'd like to use the test stage for a group of the Parse test cases both iOS and macOS. 

@mrmarcsmith  What's your thought about this? 

https://docs.travis-ci.com/user/build-stages/

> Build stages is a way to group jobs, and run jobs in each stage in parallel, but run one stage after another sequentially.
> 
> In the simplest and most common use case, you can now make one job run only if several other, parallel jobs have completed successfully.

Reply:
But it’s running the same tests as the circle CI, why run them twice?

It also prevents the Jazzy docs being published and the Cocoa Pods which probably makes sense if you have super solid tests but in our case I think it would just prevent us moving forward.